ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/autovpn-rr1.cfg

```diff
@@ -23,6 +23,8 @@ router path-selection
    !
    load-balance policy LB-DEFAULT-AUTOVPN-POLICY-CONTROL-PLANE
       path-group INET
+      path-group LTE
+      path-group MPLS
```

Comment:
This is actually fixing a bug where we want the full view of the policy on pathfinders! thanks